app/controllers/review_controller.rb

```diff
@@ -0,0 +1,58 @@
+class ReviewController < ApplicationController
+
+  def index
+    @reviews = Review.all
+  end
+
+  def show
+    @review = Review.find(params[:id])
+  end
```

Comment:
Same comment here as above - you don't seem to have a show page for each individual review (which probably makes sense!) so you can delete this path/route!